_i18n/en/resources/moneropedia/pruning.md

```diff
@@ -13,4 +13,4 @@ There are three ways to prune:
 
 Pruned nodes are very useful and preferable to @remote-nodes, but if possible, users should run a full node and opt for pruning only in case if necessary.
 
-The 'database pruning' feature [was added](https://github.com/monero-project/monero/pull/4843) on January 2019. See the [blog post](https://web.getmonero.org/2019/02/01/pruning.html) for more information.
\ No newline at end of file
+The 'database pruning' feature [was added](https://github.com/monero-project/monero/pull/4843) on January 2019. See the [blog post]({{ site.baseurl }}/2019/02/01/pruning.html) for more information.
```

Comment:
This link is broken in the preview.

Reply:
Link is broken because it's a blog post and old blog posts don't get created.

Reply:
The preview only shows the last 10 blog posts. So yes, in the preview this blog post doesn't exist. This PR needs to be tested locally.